pegasus/sites.v3/code.org/views/interactive_map.haml

```diff
@@ -58,101 +55,11 @@
       %a{href: "https://docs.google.com/document/d/1gySkItxiJn_vwb8HIIKNXqen184mRtzDX12cux0ZgZk/pub", target: '_blank'} More info
 
 :javascript
-  var webServiceLocation = '/promote/state/';
-
-  $(window).on("load", function() {
-    if (#{use_url}) {
-      // Check if the URL specifies a specific state. If so, load the state
-      if (window.location.pathname.replace("/promote", "") !== "") {
-        var state = window.location.pathname.replace("/promote/", "");
-        $.ajax({
-          url: webServiceLocation + state.toLowerCase(),
-          type: "get",
-        }).done(function(data) {
-          $("#infobox").html(data);
-        }).fail(function () {
-          if (window.history.replaceState) {
-            history.replaceState({}, "", "../promote");
-          }
-        });
-      }
-    }
-
-    $("#state-dropdown").change(
-        function(){
-
-          var abbr = $('#state-dropdown option:selected').val();
-          console.log(abbr)
-          if (abbr != '') {          // Load the selected state's information into the infobox
-            $.ajax({
-              url: webServiceLocation + abbr.toLowerCase(),
-              type: "get",
-            }).done(function(data) {
-              $("#infobox").html(data);
-            }).fail(function () {
-              if (#{use_url}) {
-                window.location = "/promote"
-              }
-            });
-          }
-        }
-      )
+  var useUrl = #{ use_url };
```

Comment:
Maybe note somewhere at top of combined JS file that it expects these to be defined?

Reply:
Good idea.  Done.